launch-dedicated.bat

```diff
@@ -0,0 +1,17 @@
+@echo on
+
+set Name="Dedicated Server"

```

Comment:
But with the name of the server, as long as it's in quotes it will not be chopped off. Take Crossroads Release servers as an example.